admin.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Mercator\GUI;
+
+use Mercator\Mapping;
+
+const VERSION = '0.1';
+
+add_action( 'admin_menu', __NAMESPACE__ . '\\menu_item' );
+add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\enqueue_scripts' );
+add_action( 'wp_ajax_mercator-gui', __NAMESPACE__ . '\\handle_ajax' );
+
+function menu_item() {
+	add_options_page(
+		__( 'Domain settings', 'mercator' ),
+		__( 'Domains', 'mercator' ),
+		'manage_options',
+		'mercator-gui',
+		__NAMESPACE__ . '\\settings_page'
+	);
+}
+
+function settings_page() {
+	include 'views/settings.php';
+}
+
+function enqueue_scripts( $hook ) {
+	if ( 'settings_page_mercator-gui' !== $hook ) {
+		return;
+	}
+
+	// Initial data object, for fast loading fun times
+	$data = array(
+		'api'   => admin_url( '/admin-ajax.php' ),
+		'nonce' => wp_create_nonce( 'wp_rest' ),
+		'data'  => array(
+			'aliases'  => array_map(
+				__NAMESPACE__ . '\\mapping_to_array',
+				Mapping::get_by_site( get_current_blog_id() )
+			),
+			'site'     => get_site( get_current_blog_id() ),
+			'mainSite' => get_site( get_main_site_for_network() ),
```

Comment:
The function is only present if `wp-multi-network` is activated

https://github.com/stuttter/wp-multi-network/search?utf8=%E2%9C%93&q=get_main_site_for_network&type=